app/src/main/java/ch/epfl/sdp/market/MarketActivity.java

```diff
@@ -55,13 +54,13 @@ protected void onCreate(Bundle savedInstanceState) {
         scanImg = findViewById(R.id.scanButton);
         shImg = findViewById(R.id.shieldButton);
         healthImg = findViewById(R.id.emsButton);
-        buy = findViewById(R.id.BuyButton);
+        Button buy = findViewById(R.id.BuyButton);
         money = findViewById(R.id.textMoney);
 
-        aoeImg.setOnClickListener(v -> invertCardView(v));
-        scanImg.setOnClickListener(v -> invertCardView(v));
-        shImg.setOnClickListener(v -> invertCardView(v));
-        healthImg.setOnClickListener(v -> invertCardView(v));
+        aoeImg.setOnClickListener(this::invertCardView);
```

Comment:
oh I never so this. So v -> is same like this::?

Reply:
It is a reference to a method